src/Selenium2Library/keywords/_formelement.py

```diff
@@ -180,6 +180,26 @@ def choose_file(self, locator, file_path):
                         % file_path)
         self._element_find(locator, True, True).send_keys(file_path)
 
+    def choose_multiple_files(self, locator, basePath="",  *files):
+        """Inputs the `files` into file input field found by `locator`.
```

Comment:
On documentation following pep257[1] is required. Also following pep008 is required, unless the is good reason to not to follow the pep. Good reason not to follow the pep are the examples in keyword documentation. 

[1] https://www.python.org/dev/peps/pep-0257/
[2] https://www.python.org/dev/peps/pep-0008/